src/e2e/java/uk/gov/hmcts/sscs/ccd/ReadCaseFromCcd.java

```diff
@@ -19,8 +19,14 @@
     private ReadCoreCaseDataService readCoreCaseDataService;
 
     @Test
-    public void givenACase_shouldBeSavedAndThenUpdatedIntoCcd() {
-        CaseDetails caseDetails = readCoreCaseDataService.getCcdCaseDetails("1520116198612015");
+    public void givenACaseId_shouldRetrieveCaseDetails() {
+        CaseDetails caseDetails = readCoreCaseDataService.getCcdCaseDetailsByCaseId("1520966754095462");
+        assertNotNull(caseDetails);
+    }
+
+    @Test
+    public void givenAnAppealNumber_shouldRetrieveCaseDetails() {
+        CaseDetails caseDetails = readCoreCaseDataService.getCcdCaseDetailsByAppealNumber("abcde12345");
         assertNotNull(caseDetails);
     }
 
```

Comment:
Please rename the class to include Test in the name.

Reply:
done.